app/src/main/res/layout-sw320dp/activity_login.xml

```diff
@@ -4,111 +4,112 @@
-            android:id="@+id/card1"
-            android:layout_width="150dp"
+        <LinearLayout
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_gravity="center"
-            android:padding="100dp"
-            app:cardCornerRadius="5dp"
-            android:layout_marginTop="30dp">
-            <Button
-                android:id="@+id/btn_login"
+            android:orientation="vertical">
+
+            <LinearLayout
+                android:id="@+id/buttonPanel"
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:background="@null"
-                android:layout_gravity="center"
-                android:padding="10dp"
-                android:text="@string/login_xml_blogin"
-                android:textColor="#000000"
-                android:textStyle="bold"
-                android:textSize="17sp"/>
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                android:layout_alignParentBottom="true"
+                android:layout_marginTop="15sp"
+                android:background="@drawable/greenbutton">
+
+                <Button
+                    android:id="@+id/btn_login"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:background="?attr/selectableItemBackground"
+                    android:text="LOGIN"
+                    android:textColor="@color/white"
+                    android:textSize="17sp"
+                    android:textStyle="bold" />
 
-        </android.support.v7.widget.CardView>
+            </LinearLayout>
 
-        <android.support.v7.widget.CardView
-            android:id="@+id/card2"
-            android:layout_width="150dp"
-            android:layout_height="wrap_content"
-            android:layout_gravity="center"
-            android:padding="100dp"
-            app:cardCornerRadius="5dp"
-            android:layout_marginTop="30dp">
             <Button
-                android:id="@+id/btn_reg"
+                android:id="@+id/toreg"
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:background="@null"
-                android:layout_gravity="center"
-                android:padding="10dp"
-                android:text="@string/login_xml_bregstr"
-                android:textColor="#000000"
-                android:textStyle="bold"
-                android:textSize="17sp"/>
-        </android.support.v7.widget.CardView>
-
+                android:layout_height="wrap_content"
+                android:fontFamily="sans-serif"
+                android:gravity="center"
+                android:background="?attr/selectableItemBackground"
+                android:text="New User? Register Here."

```

Comment:
Can we please extract this into a String resource?

Reply:
Done.